native/packages/react-native-bpk-component-phone-input/src/BpkDialingCodeList.android.js

```diff
@@ -18,25 +18,18 @@
 
 /* @flow */
 
-import { View, StyleSheet, FlatList } from 'react-native';
 import React from 'react';
-import { spacingMd, colorGray100 } from 'bpk-tokens/tokens/base.react.native';
-import BpkDialingCodeListItem from './BpkDialingCodeListItem';
+import BpkFlatList, {
+  BpkFlatListItem,
+  BpkFlatListItemSeparator,
+} from 'react-native-bpk-component-flat-list';
+import BpkFlag from './BpkFlag';
```

Comment:
Im confused - why were we not using `BpkFlag` before? I cant see it being added here...

Reply:
We were using it inside the list item component, which has been removed in favour of consuming `BpkFlatListItem`. 

It's in use at this higher level because it adds styling to the image passed in by the consumer, which we want to preserve (it sizes the image and adds a border, which is something we want here).